packages/account/src/test-utils/launchNode.test.ts

```diff
@@ -38,14 +38,36 @@ describe('launchNode', () => {
   });
 
   it('cleanup kills the started node', async () => {
-    const { cleanup, url } = await launchNode();
+    const killSpy = vi.spyOn(process, 'kill');
+
+    const { cleanup, url, pid } = await launchNode();
     expect(await fetch(url)).toBeTruthy();
 
     cleanup();
 
+    expect(killSpy).toHaveBeenCalledTimes(1);
+    // adding a minus prefix kills the process group
+    expect(killSpy).toHaveBeenCalledWith(-pid);
+
     await waitUntilUnreachable(url);
   });
 
+  /**
```

Comment:
I wonder if both tests shouldn't be just one, mocking both `kill` and `spawn` methods simultaneously and ensuring both are called when as expected.